cypress/integration/channel/add_users_to_channel_spec.js

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] in dicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 5]*/
+
+describe('Add user to a channel', () => {
+    it('CS15445 Join/leave messages (including combined): Usernames are links, open profile popovers', () => {
+        // # Login as 'user-1' and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+
+        // # Select 'doloremque' channel on the left hand side
+        cy.get('#sidebarItem_saepe-5').click();
+
+        // * 'doloremque' shoul be visible at the top of the center pane
+        cy.get('#channelHeaderTitle').should('contain', 'doloremque');
+
+        // # Then click it to access the drop-down menu
+        cy.get('#channelHeaderTitle').click();
+
+        // * The dropdown menu of te channel header should be visible;
+        cy.get('#channelHeaderDropdownMenu').should('be.visible');
+
+        // # Click 'Add Members'
+        cy.get('#channelAddMembers').click();
+
+        // * Assert that modal appears
+        cy.get('#addUsersToChannelModal').should('be.visible');
+
+        // # Wait 1 second
+        cy.wait(1000);// eslint-disable-line
+
+        // # Click the first row clickable of the modal to select a user
+        cy.get('#userAddList').first().click();
+
+        // # Click the button "Add" to add this user to 'doloremque' channel
+        cy.get('#saveItems').click();
+
+        cy.getLastPostId().then((id) => {
+            // # On the system message advising a user was added to the channel, click the username link
```

Comment:
I suggest to partially verify text content of system message, like:
``cy.get(`#postMessageText_${id}`).should('contain', 'added to the channel by you');``

While to other instance as,
``cy.get(`#postMessageText_${id}`).should('contain', '2 others were added to the channel by you');``